src/Mobs/Enderman.cpp

```diff
@@ -5,9 +5,60 @@
 #include "Enderman.h"
 #include "../Entities/Player.h"
 #include "../LineBlockTracer.h"
+#include "../BlockInfo.h"
 
+bool FindTeleportDestination(cWorld *a_World, Vector3i a_Centre, const int a_HalfCubeWidth, const int a_HeightRequired, const unsigned int a_NumTries, Vector3i &a_Destination)
```

Comment:
This could also use a `cBoundingBox` for example to define asymmetric areas. For whatever reason somewhat would use that. Maybe even reveal this to the API

Reply:
Alrighty I sorted out a cBoundingBox interface and might have successfully exported it to the API but that's all magic to me so I'm not sure.